src/domain/landingPage/landingPageHero/LandingPageHero.tsx

```diff
@@ -32,6 +34,77 @@ const LandingPageHero: React.FC<Props> = ({ landingPage }) => {
     window.open(buttonUrl || '', '_self');
   };
 
+  const width = React.useMemo(() => {
+    const getTextMaxWidth = (): number | undefined => {
```

Comment:
This return type might be little bit redundant. Typescript actually gives more explicit type implicitly here :) (400 | 560 | undefined)

Reply:
<img width="387" alt="Screenshot 2020-09-28 at 11 35 15" src="https://user-images.githubusercontent.com/15219142/94409376-c060ab80-017e-11eb-8089-1046dfdd46c1.png">

Reply:
I usually like to have explicit return types to make refactoring little bit easier. But maybe it's not needed here

Reply:
Yeah me too, no right answer here